autogpt_platform/docker-compose.platform.yml

```diff
@@ -1,20 +1,4 @@
 services:
-  postgres:
```

Comment:
the reason i didn't remove postgres entirely was so that ppl who are on non docker or used to this set up of just spinning up pg could continue to use it without yet another change to their local dev. So if you are going to remove this: 
1) please update the .env.examples to include the new pg url
2) please update the readmes for non docker to spin up just the pg inside docker compose